### PR TITLE
Fix format alignment for cluster info trace

### PR DESCRIPTION
**Pull request by HaoranYi**
#### Problem
cluster info trace is out of format alignment. 

```
    IP Address        |Age(ms)| Node identifier                              | Version |Gossip|TPUvote| TPU  |TPUfwd| TVU  |TVUfwd|Repair|ServeR|ShredVer
    ------------------+-------+----------------------------------------------+---------+------+------+-------+------+------+------+------+------+--------
    127.0.0.1       me|     2 | HWYUhPzXKzN8iED7c4C1sXUjJLnkwnYNP44ntSFHq7vS |    -    | 1026 | 58504| none | 58503| 58501| 58502| 58505| 58508| 0
    Nodes: 0
    Spies: 1

    RPC Address       |Age(ms)| Node identifier                              | Version | RPC  |PubSub|ShredVer
    ------------------+-------+----------------------------------------------+---------+------+------+--------
    127.0.0.1       me|     2 | 12UTcgCrhCBz8u7rCYpK4LaLFY5WaAZHqyn3nGhMFe7d |    -    | 23164| 6231 | 0
    RPC Enabled Nodes: 1
```

```
    IP Address        |Age(ms)| Node identifier                              | Version |Gossip|TPUvote| TPU  |TPUfwd| TVU  |TVUfwd|Repair|ServeR|ShredVer
    ------------------+-------+----------------------------------------------+---------+------+-------+------+------+------+------+------+------+--------
    127.0.0.1       me|     1 | 12UTcgCrhCBz8u7rCYpK4LaLFY5WaAZHqyn3nGhMFe7d |    -    | 1026 |  63532| none | 63531| 63529| 63530| 63533| 63536| 0
    Nodes: 0
    Spies: 1

    RPC Address       |Age(ms)| Node identifier                              | Version | RPC  |PubSub|ShredVer
    ------------------+-------+----------------------------------------------+---------+------+------+--------
    127.0.0.1       me|     2 | 12UTcgCrhCBz8u7rCYpK4LaLFY5WaAZHqyn3nGhMFe7d |    -    | 23164| 6231 | 0
    RPC Enabled Nodes: 1
```

#### Summary of Changes
Fix the alignment. 

Fixes #

### CODE REVIEW

File: gossip/Cargo.toml [hunk 1]
```diff
@@ -17,6 +17,7 @@ crossbeam-channel = "0.5"
 flate2 = "1.0"
 indexmap = { version = "1.8", features = ["rayon"] }
 itertools = "0.10.3"
+regex = "1"
```
Comment (CriesofCarrots) [inline gossip/Cargo.toml:20]: Can you please move this to dev-dependencies, since it's only used in a test?

Comment (HaoranYi) [reply gossip/Cargo.toml]: Sure.

File: gossip/Cargo.toml [hunk 2]
```diff
@@ -48,6 +48,7 @@ solana-vote-program = { path = "../programs/vote", version = "=1.11.0" }
 thiserror = "1.0"
 
 [dev-dependencies]
+regex = "1"
```
Comment (taozhu-chicago) [inline gossip/Cargo.toml:51]: +1 @CriesofCarrots for pointing this out.

**Review by taozhu-chicago — APPROVED** [review]
lgtmm

**Pull request merged**